SrfGen/NHM/fault_realisation_selection.py

```diff
@@ -0,0 +1,69 @@
+import math
```

Comment:
Can you add a header comment?

1) What it does
2) Translated from something.m
etc